components/Home/Hero.vue

```diff
@@ -1,221 +1,93 @@
 <template>
-  <div>
-    <div class="flex justify-center h-0">
-      <section
-        class="
-          container
-          items-center
-          flex-col flex
-          relative
-          z-2
-          top-48
-          max-w-lg
-          md:max-w-2xl
-        "
-      >
+  <section class="mx-auto min-h-screen flex flex-col justify-evenly text-center relative">
+    <ul class="flex flex-col md:flex-row justify-evenly">
+      <li v-for="text in headingText" :key="text">
         <h1
-          data-text="SDG"
-          class="text-6xl sm:text-7xl md:text-9xl xl:text-11xl antialiased py-4 text-center"
+          class="text-5xl lg:text-7xl antialiased mx-2 my-2 text-center tracking-widest"
         >
-          SDG
+          {{ text }}
         </h1>
-        <p class="antialiased text-shadow-md text-center px-4 text-lg lg:text-xl">
-          <span class="font-bold text-xl md:text-3xl mb-8 font-display block">
-            Club Of ProgrammerS IIT (BHU), Varanasi</span
-          >
-          We at Software Development Group are a bunch of designers and developers who aim to
-          encourage the development of technology and innovation in IIT (BHU)
-          campus (and beyond) by learning, sharing knowledge and solving
-          problems.
-        </p>
-      </section>
-    </div>
-    <section id="box" class="flex flex-col overflow-hidden w-full h-screen">
-      <div
-        v-for="index in 27"
-        :key="index"
-        class="
-          row
-          transform
-          -rotate-30
-          relative
-          flex
-          p-0
-          whitespace-nowrap
-          text-6xl
-          -top-110
-        "
-      >
-        <div class="flex">
-          <Icon v-for="name in iconRow1" :key="name" :name="name" />
-        </div>
-        <div class="flex">
-          <Icon v-for="name in iconRow2" :key="name" :name="name" />
-        </div>
-      </div>
-    </section>
-  </div>
+      </li>
+    </ul>
+    <figure>
+      <h2 class="text-4xl md:text-5xl lg:text-6xl font-bold">
+        Software Development Group
+      </h2>
+      <figcaption class="text-xl font-medium tracking-widest">Club of Programmers, IIT (BHU) Varanasi</figcaption>
```

Comment:
The vertical spacing between this and the text below is bit large, can be reduced.

Reply:
We have to fill the complete screen that's why the spacing is necessary any other alternative for it?

Reply:
In that case, we can go ahead with this.